### PR TITLE
issue #809

### PR DESCRIPTION
**Pull request by maburg**
2 files adapted with input fields for search2.
Should solve #809 .

### CODE REVIEW

File: my/XRX/src/mom/app/search/search.xqm [hunk 1]
```diff
@@ -195,7 +195,7 @@ declare function search:query-string-scope($metadata-charter-db-base-collection-
 (: the basic full text query string :)
 declare function search:term-query-string() as xs:string {
 
-    concat("$context//cei:text[ft:query(.,'", $search:q, "',$search:options)]")
+    concat("$context//cei:text[ft:query(./descendant-or-self::node(),'", $search:q, "',$search:options)]")
```
Comment (StephanMa) [inline my/XRX/src/mom/app/search/search.xqm]: Cave: there was an error, which crashes the search if there is corrupted data. By changing it back to descendant-or-self 

See #352

Comment (GVogeler) [reply my/XRX/src/mom/app/search/search.xqm]: Good that you're pointing to the old issues! The commit is related to #44 as well. And tries to solve #424 . In both we have examples how the solution should work.

Comment (maburg) [reply my/XRX/src/mom/app/search/search.xqm]: Testing was fine!

Comment (maburg) [reply my/XRX/src/mom/app/search/search.xqm]: https://dev.monasterium.net/mom/search2?q=%22Papst+Gregor%22&sort=date&context=AT-DOZA works!
But you are right (old normal) search returns jetty error!

Comment (GVogeler) [reply my/XRX/src/mom/app/search/search.xqm]: Can we already replace old search with search2 in general?

Comment (maburg) [reply my/XRX/src/mom/app/search/search.xqm]: Done!

**Comment by GVogeler** [comment]
solves #177 
Check https://github.com/icaruseu/mom-ca/milestone/33 for further issues in search2

**Comment by GVogeler** [comment]
The current solution does not constrain search by fond:
Putting "Schutz Schirm" into the search slot of GLA Karlsruhe, Allerheiligen (http://monasterium.net/mom/DE-GLAK/GLAK_34/fond) returns http://monasterium.net/mom/search?q=Schutz&context=DE-GLAK%3BGLAK_34 which includes results from"Quellen der Stadt Köln ..." and other collections and fonds. @maburg any idea, why this happens?

**Comment by GVogeler** [comment]
The search constrain by collection can be expressed in the URL via the paramter `col`. The current solution uses the parameter `context`. http://monasterium.net/mom/search?q=Schutz&col=DE-GLAK, for instance constrains the search to the GLA Karlsruhe, while http://monasterium.net/mom/search?q=Schutz&col=DE-GLAK;GLAK_34 returns no resault, although the term `Schutz` is part of the fonds "DE-GLAK/GLAK_34"

**Pull request merged**